handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -1283,13 +1283,35 @@ private boolean ignoreException(Throwable t) {
      *                  {@code true} if the {@link ByteBuf} is encrypted, {@code false} otherwise.
      * @throws IllegalArgumentException
      *                  Is thrown if the given {@link ByteBuf} has not at least 5 bytes to read.
+     * @deprecated use {@link #isEncrypted(ByteBuf, boolean)}.
      */
+    @Deprecated
     public static boolean isEncrypted(ByteBuf buffer) {
+        return isEncrypted(buffer, true);
```

Comment:
@chrisvest I was even wondering if the default should just be `false` but this might "break" some use-cases. WDYT ?

Reply:
Default protocols are TLSv1.2 and TLSv1.3 more or less everywhere these days, so my initial thought was to remove the SSLv2 support. I think that puts me in the "default `false`" camp.